Main.gd

```diff
@@ -0,0 +1,54 @@
+extends Node
+
+var rods # all the radioactive rods in the scene
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	rods = get_tree().get_nodes_in_group("rods")
+	
+
+# Called every frame. 'delta' is the elapsed time since the previous frame.
+func _process(delta):
+	var player_translation = $Player.translation
+	
+	var voxel_text = ""
+	var radiation_value = 0
+	# iterate through all rods 
+	for rod in rods:
+		# find player x,y,z with respect to the rod
+		var x = floor(rod.center_x + player_translation.x - rod.translation.x)
+		var y = floor(rod.center_y + player_translation.y - rod.translation.y)
+		var z = floor(rod.center_z + player_translation.z - rod.translation.z)
+	
+		# get the data of the voxel we are in
+		var index = -1
+		# check if we are in the voxel map
+		if x >= 0 and x <= 19 and y >= 0 and y <= 19 and z >= 0 and z <= 19:
```

Comment:
Magic numbers - let's make the mesh size a variable that we might learn on load time

Reply:
Made the mesh size into a variable, but didn't edit `points.csv` to make the program load the mesh size when the game is booted up. I can add that functionality a bit later though, when I know more about the format of the `.csv` files of the radiation maps.